src/Components/IncomeBlock/IncomeQuestion.js

```diff
@@ -241,11 +232,13 @@ const IncomeQuestion = ({
 
     return (
       <>
-        <QuestionQuestion>
-          <FormattedMessage id={formattedMsgId} defaultMessage={formattedMsgDefaultMsg} />
-          {getIncomeStreamNameLabel(allIncomeSources[index].incomeStreamName)}
-        </QuestionQuestion>
-        <div className="income-block-textfield">
+        <div className="margin-bottom">
```

Comment:
That way, we don't accidentally interfere with CSS in another part of the app.
```suggestion
        <div className="income-margin-bottom">
```

Reply:
This messes up a lot of the styling that I did and will require more time to go back and fix since it stems from the `Textfield`'s `margin-bottom` property.